cmd/vcluster/context/context.go

```diff
@@ -98,50 +19,11 @@ type ControllerContext struct {
 	CurrentNamespace       string
 	CurrentNamespaceClient client.Client
 
-	Controllers map[string]bool
+	Controllers sets.String
 	Options     *VirtualClusterOptions
 	StopChan    <-chan struct{}
 }
 
-var ExistingControllers = map[string]bool{
```

Comment:
moved this to new file: `enable_controllers.go` and unit-tested the functionality